app/views/shared/_intercom.html.erb

```diff
@@ -1,12 +1,18 @@
 <% @app_id = EnvironmentCredentials.dig(:intercom, :app_id) %>
 
+<button type="button" class="open-intercom button button--small button--green button--intercom">
```

Comment:
The `open-intercom` class does not add any styles, it enables intercom to open on click. 

This is a class because it is being used on the [tax questions public page](https://demo.getyourrefund.org/en/tax-questions), so on that page there would be two intercom buttons (aka why it's not an `id`).